integration-tests/src/test/java/org/apache/fineract/integrationtests/HookIntegrationTest.java

```diff
@@ -98,12 +93,11 @@ public void shouldSendOfficeCreationNotification() {
         } finally {
             this.hookHelper.deleteHook(hookId.longValue());
         }
-
     }
 
     @Test
     public void createUpdateAndDeleteHook() {
-        final String payloadURL = "http://echo-webhook.herokuapp.com:80/Z7RXoCBdLSFMDrpn?";
+        final String payloadURL = "http://echo-webhook.herokuapp.com:80/Z7RXoCBdLSFMDrpn/";
```

Comment:
Weird host this Heroku URL is failing in Postman with trailing / but working in Retrofit with trailing /.